code/game/turfs/flooring/flooring.dm

```diff
@@ -356,3 +358,14 @@
 	build_type = null
 	flags = TURF_ACID_IMMUNE | TURF_CAN_BREAK | TURF_REMOVE_CROWBAR
 	color = "#00ffe1"
+
+/decl/flooring/snow
+	name = "snow"
+	desc = "Let is sno-ow... Let is snow..."
```

Comment:
Spelling.

Reply:
'Iet it snow, let it snow, let it snow' is the lyric in English.